src/service-providers/active_campaign/ProviderSidebar.js

```diff
@@ -124,14 +171,68 @@ const ProviderSidebarComponent = ( {
 		}
 	}, [ stringifiedLayoutDefaults.length ] );
 
-	if ( ! inFlight && 'publish' === status ) {
+	const onChangeSendTo = async ( labels, type = 'list' ) => {
+		const isList = 'list' === type;
+		const selectedLabel = labels[ 0 ];
+		const items = isList ? [ ...lists ] : [ ...segments ];
+		const selectedItem = items.find( item => item.label === selectedLabel );
+		const metaKey = isList ? 'ac_list_id' : 'ac_segment_id';
+		const updatedMeta = {};
+		updatedMeta[ metaKey ] = selectedItem?.id || '';
+		updateMeta( updatedMeta );
+		return selectedItem;
+	};
+
+	if ( ! inFlight && ( 'publish' === status || 'private' === status ) ) {
 		return (
 			<Notice status="success" isDismissible={ false }>
 				{ __( 'Campaign has been sent.', 'newspack-newsletters' ) }
 			</Notice>
 		);
 	}
 
+	if ( ! lists?.length ) {
+		return (
+			<div className="newspack-newsletters__loading-data">
+				{ __( 'Retrieving ActiveCampaign data…', 'newspack-newsletters' ) }
+				<Spinner />
+			</div>
+		);
+	}
+
+	const renderSelectedSummary = () => {
+		return selectedList ? (
+			<p
+				dangerouslySetInnerHTML={ {
+					__html: sprintf(
+						// Translators: %1$s is the number of members, %2$s is the item name, %3$s is the item type, %4$s is the subitem name and type (if any).
+						__(
+							'This newsletter will be sent to <strong>%1$s</strong> in the <strong>%2$s</strong> %3$s%4$s.',
+							'newspack-newsletters'
+						),
+						! selectedSegment && selectedList?.count // AC doesn't provide contact counts for segments, so we only want to show a count if a list is selected without a segment.
+							? sprintf(
+									// Translators: %d is the number of contacts in the list.
+									_n( '%d contact', '%d contacts', selectedList.count, 'newspack-newsletters' ),
+									selectedList.count.toLocaleString()
+							  )
+							: __( 'all contacts', 'newspack-newsletters' ),
+						selectedList.name,
+						selectedList.typeLabel.toLowerCase(),
+						selectedSegment
+							? sprintf(
+									// Translators: %1$s is the parent item name, %2$s is the parent item type.
+									__( ' who are part of the %1$s %2$s', 'newspack-newsletters' ),
```

Comment:
here you are also breaking a phrase into separate translatable strings.

If there are many conditions that can change the elements of the text, bring the checks up in the code and make one completely independent string for each possible situation, even if they have parts that will be identical and you'll have some duplicate code. That's the best practice for i18n.

The main idea is that a phrase should be self contained and have the complete sentence. No "ifs" in the middle. This not only makes the job easier for translators but also gives them flexibility to rephrase things as needed, to adapt and actually localize the string - and not only do a literal translation